RecoEgamma/ElectronIdentification/python/Identification/mvaElectronID_PHYS14_PU20bx25_nonTrig_V1_cff.py

```diff
@@ -0,0 +1,97 @@
+from PhysicsTools.SelectorUtils.centralIDRegistry import central_id_registry
```

Comment:
this file was recovered with a commit "Brought back 2014 Electron MVA for backwards compatibility".
Please clarify in a bit more detail which backwards compatibility is implied here.
Can this file be even used with existing producers in this release?

Reply:
It's fully backwards compatible. It can be added to the value map produced https://github.com/cms-sw/cmssw/blob/master/RecoEgamma/ElectronIdentification/python/ElectronMVAValueMapProducer_cfi.py and produces the same values as the previous config file which was deleted.

Reply:
I think it's interesting to keep for comparisons. It's the only Run2 MVA ID which doesn't include conversion rejection yet. But if you think there is no place for it here I'll just keep it privately.

Reply:
if it's useful, it should be enabled in the unit tests

Reply:
IIUC, this was removed in 10_1_X.
Did EGM group find some new use cases for this ID?

Reply:
No, but for me it was useful to compare with HGCal since there is no conversion rejection there. But I see that I should better remove it here and keep it privately.